main.go

```diff
@@ -64,7 +64,13 @@ func main() {
 	go http.Serve(l, nil)
 
 	u := fmt.Sprintf("http://%v/", *listenAddr)
-	if !*doBrowser || !openBrowser(u) {
+	browserOpened := false
```

Comment:
I think this would be simpler as

```
if *doBrowser {
  u := fmt.Sprintf("http://%v/", *listenAddr)
  if openBrowser(u) {
    fmt.Printf("Open your web browser to %v\n\n", u)
  }
}
```

Reply:
sure. but the user has to know what address to open even if they don't want to launch a browser.